sensitive-data/README.md

```diff
@@ -0,0 +1,5 @@
+# Bundle with sensitive data
+
+This bundle demonstrates how porter works with bundle that contains sensitive
+data. It requires users to set up a secret store in their porter configuration
```

Comment:
Can you provide steps for what they should configure (it's okay to link to the quickstart or existing doc) and then include a section that has them try out the bundle?

You can look at https://github.com/getporter/examples/tree/main/otel-jaeger for the amount of detail that we want to include in the example readmes. Essentially someone should be able to figure out how to configure porter and then run the example bundle.